docs/course_materials/exercises/test_exercises/README_add_tests.md

```diff
@@ -0,0 +1,62 @@
+# Adding test files for Linux
```

Comment:
The name of the file could be just "how_to_add_tests.md"
same for "how_to_run_tests.md"